src/js/integrations/devicecontext.ts

```diff
@@ -47,7 +48,14 @@ export class DeviceContext implements Integration {
         event.user = nativeUser;
       }
 
-      const nativeContext = native.context;
+      let nativeContext = native.context;
+      if (AppState.currentState !== 'unknown') {
+        nativeContext = nativeContext || {};
+        nativeContext.app = {
+          ...nativeContext.app,
+          in_foreground: AppState.currentState === 'active',
```

Comment:
Doesn't it need to be set to `"false"` at some point too? Or are you passing the `nativeContext` to the native SDKs and they set it for you?

Reply:
The `in_foreground` is merged into what I get from native SDKs. As I think the value that RN reports is important for RN devs.

If the `AppState.currentState === 'active'` => `true`, else `false`.

Thanks for the comment I've added a test to make it clear how it works.

Reply:
Ahh, I see now, gotcha. Thanks!